docs/geode-native-docs/introduction/client-configurations.html.md.erb

```diff
@@ -157,7 +157,7 @@ The following libraries are external dependencies of the client library, `libgfc
 
 If you plan on using SSL in your Geode client and server deployment, you will need to download and install OpenSSL.
 
-The client requires OpenSSL 1.0.1t or later. For Windows platforms, you can use either the regular or the OpenSSL "Light" version.
+The client requires OpenSSL version 1.0.2. For Windows platforms, you can use either the regular or the OpenSSL "Light" version.
```

Comment:
I still feel like it would be better to save that "For Windows..." statement for a common area about where to get OpenSSL.